src/Queue/BunnyTickingConsumer.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Skrz\Bundle\BunnyBundle\Queue;
+
+use Bunny\Channel;
+use LogicException;
+
+/** @author Lukas Senfeld <skrz@senfeld.net> */
+abstract class BunnyTickingConsumer extends BunnyConsumer
+{
+
+	protected ?float $tickSeconds = null;
```

Comment:
Tohle je úplně stejný jako v BunnyConsumer (ale tam nevím, jestli to vůbec má být)

Reply:
odstraneno z BunnyConsumeru patri to jenom sem